src/server/test/web/readingsBarGroupQuantity.js

```diff
@@ -207,8 +207,22 @@ mocha.describe('readings API', () => {
 
                 // Add BG13 here
 
-                // Add BG14 here
+                mocha.it('BG14: 1 day bars for 15 + 20 minute reading intervals and quantity units with +-inf start/end time & kWh as lbs of CO2 & chained & reversed', async () =>{
+                    //loads data into database
+                    await prepareTest(unitDatakWh, conversionDatakWh, meterDatakWhGroups, groupDatakWh);
+                    //gets unit ID
+                    const unitId = await getUnitId('pound of CO₂');
+                    //load data from mcsv file
+                    const expected = await parseExpectedCsv('src/server/test/web/readingsData/expected_bar_group_ri_15-20_mu_kWh_gu_lbsCO2_st_-inf_et_inf_bd_1.csv');
```

Comment:
The expected file needs to be downloaded from the Dev Docs and added in the appropriate place in the repository. Direction are here: https://github.com/OpenEnergyDashboard/DesignDocs/blob/main/testing/testing.md#documentation-on-how-testing-data-is-created-and-used.